build.yaml

```diff
@@ -1230,8 +1235,8 @@ libs:
   deps:
   - grpc++_test_util
   - grpc_test_util
-  - grpc++
   - grpc
+  - grpc++
```

Comment:
The dependencies in build.yaml need to come in reverse order. (grpc++  comes before grpc)
(many occurences).